docs/README.md

```diff
@@ -284,14 +284,18 @@ configured [here](#set-up-an-aws-account)
 ℹ Project generated!
 ```
 
-> Booster's new: commands follows this structure
+> Booster CLI commands follow this structure: `boost <subcommand> [<flags>] [<parameters>]`. 
+> Let's break down the command we have just executed:
 >
-> - `boost` is the Booster CLI
-> - `new:<resource>` new is a CLI command, :project tells booster the kind of resource
-> - `boosted-blog` is a parameter for the `new:project` command
->   project name
-
-The `new:project` command generates some scaffolding for you. The project name will be the
+> - `boost` is the Booster CLI executable
+> - `new:project` is the "subcommand" part. In this case, it is composed of two parts separated by a colon. 
+> The first part, `new`, means that we want to generate a new resource. The second part, `project`, indicates which
```

Comment:
if "new" means "generate" maybe we should consider renaming it?

Reply:
Yes! I thought exactly the same Javi. 

Reading the documentation made me realize that.

Ticket: https://agilemonkeys.atlassian.net/browse/BOOST-868

Reply:
Agree 💯 **%**